composer.lock

```diff
@@ -0,0 +1,2337 @@
+{
```

Comment:
Since it's a package we don't need `composer.lock` file here. Could you please remove it?

Reply:
Ah yes, I'm used to commiting that for projects.